buildSrc/src/main/java/Versions.kt

```diff
@@ -0,0 +1,86 @@
+object Versions {
```

Comment:
Do we need to use `src/mainjava` for this since it's a Kotlin file? We never ended up changing the path of the main codebase for legacy issues, but for new code I think we can start using `src/main/kotlin` instead.

Reply:
Good point, will make the change